docs/source/models/other_models/alzheimers_population/index.rst

```diff
@@ -283,9 +328,294 @@ This is the population we pull from GBD using get_population. Thus,
+females aged 70--74 who enter the BBBM-AD state in 2025 is
+approximately
+
+.. math::
+
+  \begin{align*}
+  \gamma_{g,t}
+  &\approx 1 - \exp \left(-\Delta
+    \cdot m_{(F,70) + 3.5,\, 2025 + 3.5} \right) \\
+  &= 1 - \exp \left(-\Delta \cdot m_{(F,75),\, 2029} \right).
+  \end{align*}
+
+Note that since we have estimates of mortality rates for 5-year age
+groups and single years, we have rounded to the nearest age group and
+year. For example :math:`(F,70) + 3.5` represents females aged
+73.5--78.5 (i.e., :math:`[73.5, 78.5)`), so we round to the nearest age
+group of 75--79 (i.e., :math:`[75, 80)`). With :math:`\Delta = 7` years
+and :math:`w = 5` years, :math:`g+ \Delta/2` should always get rounded
+to :math:`g + 5`. For the year, 2025 + 3.5 = 28.5, and, since this is
+right at the year's midpoint, I've arbitrarily rounded up instead of
+down.
+
+.. note::
+
+  We can get a better approximation of :math:`\gamma_{g,t}` by making a
+  more careful approximation of the integral in the exponent:
+
+  .. math::
+
+    \begin{align*}
+    \Delta \cdot \bar m_{g,t}
+      \approx \frac{w}{2} \cdot m_{g,\, t}
+    &+ w \cdot m_{g+w,\, t+w} \\
+    &+ \dotsb \\
+    &+ w \cdot  m_{g+(n-1)w,\, t+(n-1)w} \\
+    &+ \left(\frac{w}{2} - \frac{r^2}{2w} + r\right)
+      \cdot  m_{g+nw,\, t+nw} \\
+    &+ \frac{r^2}{2w}
+      \cdot  m_{g+(n+1)w,\, t+(n+1)w}.
+    \end{align*}
+
+  Note that the weights add up to :math:`\Delta`, showing that this is
+  is a refinement of the approximation :math:`\Delta \cdot m_{g +
+  \frac{\Delta}{2} ,\, t + \frac{\Delta}{2}}`.
+
+  With :math:`\Delta = 7` years and :math:`w = 5` years, we have
+  :math:`n = 1`, so there are only three terms in the sum, corresponding
+  to :math:`g`, :math:`g + 5`, and :math:`g + 10`, and the coefficients
+  for these three terms are the three special cases in the above sum
+  (the generic coefficient of :math:`w` never appears). Using our
+  running example,
+
+  .. math::
+
+    \gamma_{(F, 70),\, 2025}
+    \approx 2.5 \cdot m_{(F, 70),\, 2025}
+    + 4.1 \cdot m_{(F, 75),\, 2030}
+    + 0.4 \cdot m_{(F, 80),\, 2035}.
+
+Entrance rate into the BBBM state in the simulation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Finally, as noted above, the rate at which real-world people in
+demographic group :math:`g` are entering the BBBM-AD state at time
+:math:`t` is approximately :math:`\dot B_{g,t}^\text{AD} \cdot
+\frac{1}{1 - \gamma_{g,t}}`. Multiplying by the model scale :math:`S`,
+the rate at which we want to add simulants into the BBBM-AD state is
+then
+
+.. math::
+
+  \lambda_{g,t} = S \cdot \dot B_{g,t}^\text{AD}
+  \cdot \frac{1}{1 - \gamma_{g,t}}.
+
+If :math:`t` is a step time of the simulation, the number of simulants
+to add at time :math:`t` is then
+
+.. math::
+
+  \text{Number of simulants to add at time t}
+  = \lambda_{g,t} \cdot \Delta t,
+
+where :math:`\Delta t` is the step size of the simulation (currently
```

Comment:
Is it bad to use $\Delta t$ for the step size when I'm also using $\Delta$ for the duration?